recipes/kikuchipy/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "kikuchipy" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5751de9db4eaef72522022134df945a5c221c83e5a22d9119d0e2c4b3338aa08
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install -e ."
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - dask
+    - hyperspy >=1.5.2
+    - h5py
+    - matplotlib
+    - numpy >=1.17
+    - pyxem >=0.10
+    - scikit-image
+    - scikit-learn
+    - scipy
+
+test:
+  requires:
+    - pytest >=5.3.2
+  imports:
+    - kikuchipy
+    - kikuchipy.io
+    - kikuchipy.signals
+    - kikuchipy.util
+  commands:
+    - pytest
```

```suggestion
    - pytest --pyargs kikuchipy
```
is very likely the command you want to run.

---

I tested this recipe locally, and `pytest --pyargs kikuchipy` failed with:
```
OSError: No filename matches '$PREFIX/lib/python3.7/site-packages/kikuchipy/io/test/../../data/nordif/Pattern.dat'.
```
type of errors.
If this can be fixed great, if not, maybe skipping the test is better?